src/Microsoft.TemplateEngine.Cli/LocalizableStrings.Designer.cs

```diff
@@ -1043,7 +1043,29 @@ public class LocalizableStrings {
                 return ResourceManager.GetString("NoTemplatesMatchName", resourceCulture);
             }
         }
-        
+
+        /// <summary>
+        ///   Looks up a localized string similar to The list of templates was synchronized with the Optional SDK Workloads.
+        /// </summary>
+        public static string OptionalWorkloadsSynchronized
+        {
+            get
+            {
```

Comment:
Do we want to keep the same indentation/style with the rest of the file?

Reply:
fixed